e2e/utils.sh

```diff
@@ -223,37 +184,20 @@ dfx_replica() {
     local replica_port dfx_config_root
     dfx_patchelf
     determine_network_directory
-    if [ "$USE_IC_REF" ]
-    then
-        # Bats creates a FD 3 for test output, but child processes inherit it and Bats will
-        # wait for it to close. Because `dfx start` leaves child processes running, we need
-        # to close this pipe, otherwise Bats will wait indefinitely.
-        dfx replica --emulator --port 0 "$@" 3>&- &
-        export DFX_REPLICA_PID=$!
-
-        timeout 60 sh -c \
-            "until test -s \"$E2E_NETWORK_DATA_DIRECTORY/ic-ref.port\"; do echo waiting for ic-ref port; sleep 1; done" \
-            || (echo "replica did not write to \"$E2E_NETWORK_DATA_DIRECTORY/ic-ref.port\" file" && exit 1)
 
-        test -f "$E2E_NETWORK_DATA_DIRECTORY/ic-ref.port"
-        replica_port=$(cat "$E2E_NETWORK_DATA_DIRECTORY/ic-ref.port")
+    # Bats creates a FD 3 for test output, but child processes inherit it and Bats will
+    # wait for it to close. Because `dfx start` leaves child processes running, we need
+    # to close this pipe, otherwise Bats will wait indefinitely.
+    dfx replica --port 0 "$@" 3>&- &
```

Comment:
The`dfx_replica` function must be dead as well, because the `dfx replica`command doesn't exist anymore.

Reply:
Dropped!